src/components/Plugin/Plugin.tsx

```diff
@@ -136,6 +141,53 @@ export function PluginStore(props: PluginProps) {
       })
     : null;
 
+  const clonedTopActionDisabled = preIcon
+    ? cloneElement(topAction, {
+        className: 'w-6 h-6 text-tertiary cursor-pointer',
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        onClick: () => {},
```

Comment:
I want to cancel the onclick function in the element 
disabled didn't work

Reply:
What about turn the function optional and just don't passing it!?

Otherwise you can just return...

```suggestion
        onClick: () => {return;},
```
But I don't like the idea of override something to remove the function returning. I think we should just don't pass it.